### PR TITLE
Fixing runtime error on ActionPopup click & LOVE_ENDPOINT spelling error

### PR DESCRIPTION
**Pull request by Nivedit24**
* Provide a brief description of the changeset.
  * Fixing runtime error on ActionPopup click
  * Need more information what "Contact Record" click event should perform
  * Doc value was undefined when called from line 201 in ActionsPopup.svelte file
  * Fixing LOVE_ENDPOINT spelling error
* Include a screenshots if applicable

https://github.com/hcengineering/platform/assets/39114793/861c1dd6-da2c-47e7-8800-3f9262ec6b30

https://github.com/hcengineering/platform/assets/39114793/dd91c2af-f2c0-48d9-9e65-bb7ace2685ca

* Ensure that the changeset adheres to the [DCO guidelines](https://github.com/apps/dco).

<sub><a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NjZhMDhlYTJkOTA4MTUzNTY4M2MzYWIiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIiwicHJvZHVjdElkIjoiIn0.0I5Wuee9q0q895WKxoIvzorkCOjBw5sme_bd-SexI0M">Huly&reg;: <b>UBERF-7250</b></a></sub>

### CODE REVIEW

**Review by haiodo — APPROVED** [review]

File: plugins/contact-resources/src/index.ts [hunk 1]
```diff
@@ -286,7 +286,10 @@ async function kickEmployee (doc: Person): Promise<void> {
     )
   }
 }
-async function openChannelURL (doc: Channel): Promise<void> {
+async function openChannelURL(doc: Channel): Promise<void> {
+  if (doc.value === undefined) {
+    return
```
Comment (Nivedit24) [inline plugins/contact-resources/src/index.ts:291]: @haiodo How to fix the formatting CI check?

**Comment by haiodo** [comment]
`rush format` should be called locally.

File: plugins/contact-resources/src/index.ts (hunk 1 above)
Comment (haiodo) [reply plugins/contact-resources/src/index.ts:291]: Hi, `rush format` should be called locally, for entire project. and `rushx format` for individual ones.

Comment (Nivedit24) [reply plugins/contact-resources/src/index.ts:291]: `rush format` is giving an error in a file unchanged by me. `rushx format` on the current file produces no changes

**Pull request closed without merging**